src/components/Collection/List.js

```diff
@@ -17,7 +17,8 @@ const CollectionList = () => {
   }, []);
 
   async function getCollections() {
-    let response = await getAllCollections(100);
+    let response = await getAllCollections(250);
+    console.log("response", response);
```

Comment:
Do you want to leave in the logging?

Reply:
Ooops, sorry thought I pushed that `amend` commit but apparently not.  Good catch.